apps/dot-voting/contracts/DotVoting.sol

```diff
@@ -444,7 +445,8 @@ contract DotVoting is ADynamicForwarder, AragonApp {
         uint256[] storage oldVoteSupport = voteInstance.voters[msg.sender];
         bytes32[] storage cKeys = action.optionKeys;
         uint256 cKeysLength = cKeys.length;
-        require(cKeysLength <= _supports.length, "More supports given than candidates");
+        // require(cKeysLength <= _supports.length, "More supports given than candidates");
+        require(cKeysLength <= _supports.length);
```

Comment:
I don't think these `require` changes should be here

Reply:
Good catch, it was a mistake from my side, fixed now